auth/grpctransport/grpctransport.go

```diff
@@ -143,13 +143,17 @@ type InternalOptions struct {
 	// DefaultAudience specifies a default audience to be used as the audience
 	// field ("aud") for the JWT token authentication.
 	DefaultAudience string
-	// DefaultEndpoint specifies the default endpoint.
-	DefaultEndpoint string
+	// DefaultEndpointTemplate combined with DefaultUniverseDomain specifies
+	// the default endpoint.
+	DefaultEndpointTemplate string
```

Comment:
Will this also be done for mTLS in the future? If so let's rename it now as well here so we are not stuck in the future. Rather make all breaking changes at once.

Reply:
Gonna leave this for now, per offline discussion.